lib/ansible/cli/doc.py

```diff
@@ -1350,16 +1350,15 @@ def get_man_text(doc, collection_name='', plugin_type=''):
         if doc.get('deprecated', False):
```

Comment:
Minor nit, but since this is controller-side code, if you walrus the first access, you can just use the local in the boatload of subsequent accesses below.

```suggestion
        if deprecated := doc.get('deprecated', False):
```